tests/e2e/e2e_test.go

```diff
@@ -73,20 +73,22 @@ var _ = Describe("odoe2e", func() {
 			It("Should fail if user tries to create anything other than project", func() {
 				runCmdShouldPass(fmt.Sprintf("odo login -u %s -p %s", "odonoprojectattemptscreate", loginTestUserPassword))
 				session := runCmdShouldFail("odo create nodejs")
-				Expect(session).To(ContainSubstring("You dont have permission to project 'default' or it doesnt exist."))
+				Expect(session).To(ContainSubstring("You dont have permission to project"))
```

Comment:
The original message ```You dont have permission to project 'default' or it doesnt exist.``` is successfully evaluated on 4.0 cluster deployed on aws. 

But in case of prow the default name space is showing something else causing the failure. Cutting short the message would be fine.